api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/support/CasConfigurationJasyptCipherExecutor.java

```diff
@@ -67,6 +88,9 @@ private static String getJasyptParamFromEnv(final Environment environment, final
      */
     public void setAlgorithm(final String alg) {
         if (StringUtils.isNotBlank(alg)) {
+            if (ALGORITHM_BLACKLIST_SET.contains(alg)) {
+                LOGGER.warn("Configured Jasypt algorithm [{}] doesn't work for decryption due to Jasypt bug", alg);
```

Comment:
I would error out here. A blacklistd algorithm should never be allowed for use.

Reply:
I made it throw illegal argument exception out but I had to add a different protected method for the unit test to call to set the bad algorithm.